internal/dkg/actions_active.go

```diff
@@ -102,8 +102,10 @@ func (d *Process) Command(ctx context.Context, command *drand.DKGCommand) (*dran
 
 		packetToGossip.Metadata = metadata
 		errs := d.gossip(me, recipients, packetToGossip)
-		// we gossip the leavers separately - if it fails, no big deal
-		_ = d.gossip(me, afterState.Leaving, packetToGossip)
+		if len(afterState.Leaving) != 0 {
+			// we gossip the leavers separately - if it fails, no big deal
+			_ = d.gossip(me, afterState.Leaving, packetToGossip)
+		}
```

Comment:
Good catch, this finally explains the unexplainable "gossip had no recipients" warnings.